src/components/sections/Section.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Section.css'
+
+
+const Section = ({
+        text,
+        color
+    }) => {
+
+
+    async function readSections () {
+        // Reading parse objects is done by using Parse.Query
+        const parseQuery = new Parse.Query('Section')
+        try {
+            let sections = await parseQuery.find()
+            console.log(sections)
+            return true
+        } catch (error) {
+            // Error can be caused by lack of Internet connection
+            alert(`Error! ${error.message}`)
+            return false;
+        }
+    }
+
+    function Colors(props) {
+        return (
+            <div className="colors" style={{backgroundColor: props.color}}>{props.children}</div>
```

Comment:
Is it not possible to style in a CSS file? I am just curious 🧐  I have not seen this before - but sure is interesting!